client/dashboard/profile-wizard/steps/business-details.js

```diff
@@ -276,38 +277,35 @@ class BusinessDetails extends Component {
 		const revenueOptions = [
 			{
 				key: 'none',
-				label: _x( "$0 (I'm just getting started)", '$0 revenue amount', 'woocommerce-admin' ),
+				label: sprintf(
+					_x( "%s (I'm just getting started)", '$0 revenue amount', 'woocommerce-admin' ),
```

Comment:
Since this string has placeholders, a `/* Translators...` comment should be added before this line.

Reply:
Added in https://github.com/woocommerce/woocommerce-admin/pull/3324/commits/9a88072b8c8265120be44eab1af7d9b26bb1b064